lakefile.toml

```diff
@@ -3,9 +3,14 @@ defaultTargets = ["FLT"]
 
 [leanOptions]
 pp.unicode.fun = true # pretty-prints `fun a ↦ b`
-autoImplicit = false # switch off auto-implicit
-relaxedAutoImplicit = false # switch off relaxed auto-implicit
-
+autoImplicit = false # no "assume a typo is a new variable"
+relaxedAutoImplicit = false # no "assume a typo is a new variable"
+linter.style.longLine = true # no lines longer than 100 chars
+linter.oldObtain = true # no "stream of consciousness `obtain`" -- remove `weak` after bump?
+linter.style.cases = true # no `cases'` tactic -- remove `weak` after bump?
+linter.style.refine = true # no `refine'` tactic
+linter.style.multiGoal = true # no multiple active goals
+weak.linter.flexible = true # no rigid tactic (e.g. `exact`) after a flexible tactic (e.g. `simp`)
```

Comment:
And this `weak` can be removed, right?

Reply:
Yes I think so. I was randomly adding them when files which were just one big comment were failing linting because they weren't importing Mathlib.Init